apps/core/src/constants/errorMessages.constants.ts

```diff
@@ -8,3 +8,5 @@ export const GAS_BUDGET_ERROR_MESSAGES = {
     [GAS_BALANCE_TOO_LOW_ID]: 'Not enough balance to cover transaction fees.',
     [NOT_ENOUGH_BALANCE_ID]: 'Not enough balance to create the transaction.',
 };
+
+export const NO_BALANCE_GENERIC_MESSAGE = 'You might not have enough balance to pay for gas fees.';
```

Comment:
Since this is no balance message we could be more direct in this message ex. : 
"Make sure you have enough funds and then try again."
"Please verify you have enough funds and try again."